stdlib/lib2to3/fixes/fix_future.pyi

```diff
@@ -0,0 +1,9 @@
+from typing import ClassVar
+from typing_extensions import Literal, LiteralString
+
+from .. import fixer_base
+
+class FixFuture(fixer_base.BaseFix):
+    BM_compatible: ClassVar[Literal[True]]
+    PATTERN: ClassVar[LiteralString]
```

Comment:
I don't think LiteralString makes much sense in cases like this. Let's avoid using it.